Commands/settings.py

```diff
@@ -0,0 +1,101 @@
+import discord
+import asyncio
+
+from discord.ext import commands
+from .GuildData import get_guild_data, save_configs
+
+
+class Settings(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command(name="add_admin", brief="**admin-only** \n\u2003 Add a bot admin", usage="<mentioned user>", help="*admin-only* \u2003 Add a bot admin. The user will be given access to the bot commands that are labelled **admin-only**. The user will not be given the role of server admin. \n\n<mention user> \u2003 Mention the user that you want to make a bot admin.")
+    async def add_admin(self, ctx, mention):
+        guild_data = await get_guild_data(ctx.message.guild.id)
+
+        if not (ctx.message.author.guild_permissions.administrator or ctx.author.id in guild_data.bot_admins):
+            # if the user is not a bot admin or server admin
+            await ctx.send("https://gph.is/g/4w8PDNj")
+            return
+        elif not ctx.message.mentions:
+            # if the message has no mentions
+            await ctx.send("You should mention a user, foemp.")
+            return
+
+        user_id_to_add = ctx.message.mentions[0].id
+        user_name_to_add = ctx.message.guild.get_member(
+            int(user_id_to_add)).display_name
+        if ctx.message.guild.get_member(int(user_id_to_add)).guild_permissions.administrator:
+            # if the requested user is already a server admin
+            await ctx.send(f"{user_name_to_add} is a server admin, so no need to make them a bot admin.")
```

Comment:
you missed a foemp here

Reply:
also, this says admin, other lines say boss. please make sure you use the same word everywhere :)